src/headerData.json

```diff
@@ -1,164 +1,201 @@
-      "url": "https://appint.fromdoppler.net/Steps/FirstSteps",
-      "isEnabled": false,
-      "isSelected": false,
-      "idHTML": "firstSteps"
-    }, {
-      "title": "Logout ",
-      "url": "https://appint.fromdoppler.net/SignIn/SignOut",
-      "isEnabled": false,
-      "isSelected": false,
-      "idHTML": "signOut"
-    }]
+    "nav": [
+      {
+        "title": "Panel de Control",
+        "url": "https://app2.fromdoppler.com/ControlPanel/ControlPanel/",
+        "isEnabled": false,
+        "isSelected": false,
+        "idHTML": "controlPanel"
+      },
+      {
+        "title": "Centro de descargas",
+        "url": "https://app2.fromdoppler.com/DownloadCenter/DownloadCenter",
+        "isEnabled": false,
+        "isSelected": false,
+        "idHTML": "downloadCenter"
+      },
+      {
+        "title": "Primeros pasos en Doppler",
+        "url": "https://app2.fromdoppler.com/Steps/FirstSteps",
+        "isEnabled": false,
+        "isSelected": false,
+        "idHTML": "firstSteps"
+      },
+      {
+        "title": "Salir",
+        "url": "https://app2.fromdoppler.com/SignIn/SignOut",
+        "isEnabled": false,
+        "isSelected": false,
+        "idHTML": "signOut"
+      }
+    ]
+  },
+  "alert": {
+    "type": "info",
+    "message": "Posees una cuenta gratis de 500  Suscriptores. Potencia tus Campañas con los beneficios de un Plan Pago.",
+    "button": {
+      "text": "Comprar Ahora",
+      "url": "https://app2.fromdoppler.com/ControlPanel/AccountPreferences/PreUpgrade"
+    }
   },
-  "homeUrl": "https://appint.fromdoppler.net/Campaigns/Draft/",
-  "urlBase": "https://appint.fromdoppler.net/",
-  "datahubCustomerId": "1024",
   "features": {
-    "siteTrackingEnabled": true,
-    "siteTrackingActive": true,
-    "emailParameterEnabled": true,
-    "emailParameterActive": true
+    "siteTrackingEnabled": false,
+    "siteTrackingActive": false,
+    "emailParameterEnabled": false,
+    "emailParameterActive": false
   },
-  "jwtToken": "eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc1N1IjpmYWxzZSwic3ViIjoiY2Jlcm5hdEBnZXRjcy5jb20iLCJjdXN0b21lcklkIjoiMTAyNCIsImRhdGFodWJDdXN0b21lcklkIjoiMTAyNCIsImlhdCI6MTU1NjU0NTQ2MSwiZXhwIjoxNTU2NTQ3MjYxfQ.meC1sFsC7m8nGFfcK2oih1hqdKQ4Lj81O5rN_-awOcM_JEe9ddviocPbYAg64L6nC6b0a7J5xmTmiW_-MAIEIg"
-}
+  "homeUrl": "https://app2.fromdoppler.com/Campaigns/Draft/",
+  "urlBase": "https://app2.fromdoppler.com/",
+  "loginUrl": "https://app.fromdoppler.com/#/login?lang=es",
+  "notifications": [
+    "<strong>NUEVO: Comportamiento en Sitio</strong><br/><p class='text--small'>Programa un flujo automatizado de Emails que se env&#237;e de acuerdo a las p&#225;ginas que los Suscriptores hayan visitado en tu sitio. Para probarlo <a target='_self' href='https://app2.fromdoppler.com/ControlPanel/AccountPreferences/UpgradeAccount/?Plan=monthly'><strong>CONTRATA UN PLAN</strong></a>.</p>"
+  ],
+  "emptyNotificationText": "No tienes notificaciones pendientes."
```

Comment:
The map behavior for these fields is missing, right?

Reply:
Correct, it is missing in doppler BE as well.

Reply:
Yes, BE is outside this PR, but the mapping should be.